card/mod/standard/set/all/rich_html/editing.rb

```diff
@@ -187,7 +187,7 @@ def rename_referer_effect_and_option
     [effect, option]
   end
 
-  view :edit_type, perms: :update do
+  view :edit_type, cache: :never, perms: :update do
```

Comment:
ok, fair enough.  for forms we can say the burden of proof is on showing the need to cache.